### ISSUE TITLE
add home charger doesn't work

### ISSUE THREAD

**Issue by nvanggaard** [fars-fede-fire/clever_api#3]
## System Information

version | core-2023.3.6
-- | --
installation_type | Home Assistant Container
dev | false
hassio | false
docker | true
user | root
virtualenv | false
python_version | 3.10.10
os_name | Linux
os_version | 6.1.16-2-MANJARO-ARM-RPI
arch | aarch64
timezone | Europe/Copenhagen
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 4644
Installed Version | 1.31.0
Stage | running
Available Repositories | 1250
Downloaded Repositories | 16

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Dashboards</summary>

dashboards | 3
-- | --
resources | 13
views | 11
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | 7 March 2022 at 16:38
-- | --
current_recorder_run | 28 March 2023 at 20:07
estimated_db_size | 7098.55 MiB
database_engine | mysql
database_version | 10.10.2

</details>

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [previous issues](https://github.com/ludeeus/integration_blueprint/issues?q=is%3Aissue+label%3A%22Bug%22+)..

### Describe the issue

When trying to add a home charger i just get an error

### Reproduction steps

1. add from hacs
2. setup integration
3. write mail
4. get login token
5. fill login token
6. check the checkbox "add home charger"
7. click add
8. error occurs
...

### Debug logs

```text
| 2023-03-28 18:21:22.406 ERROR (MainThread) [aiohttp.server] Error handling request
homeassistant  | Traceback (most recent call last):
homeassistant  |   File "/usr/local/lib/python3.10/site-packages/aiohttp/web_protocol.py", line 433, in _handle_request
homeassistant  |     resp = await request_handler(request)
homeassistant  |   File "/usr/local/lib/python3.10/site-packages/aiohttp/web_app.py", line 504, in _handle
homeassistant  |     resp = await handler(request)
homeassistant  |   File "/usr/local/lib/python3.10/site-packages/aiohttp/web_middlewares.py", line 117, in impl
homeassistant  |     return await handler(request)
homeassistant  |   File "/usr/src/homeassistant/homeassistant/components/http/security_filter.py", line 67, in security_filter_middleware
homeassistant  |     return await handler(request)
homeassistant  |   File "/usr/src/homeassistant/homeassistant/components/http/forwarded.py", line 227, in forwarded_middleware
homeassistant  |     return await handler(request)
homeassistant  |   File "/usr/src/homeassistant/homeassistant/components/http/request_context.py", line 28, in request_context_middleware
homeassistant  |     return await handler(request)
homeassistant  |   File "/usr/src/homeassistant/homeassistant/components/http/ban.py", line 80, in ban_middleware
homeassistant  |     return await handler(request)
homeassistant  |   File "/usr/src/homeassistant/homeassistant/components/http/auth.py", line 234, in auth_middleware
homeassistant  |     return await handler(request)
homeassistant  |   File "/usr/src/homeassistant/homeassistant/components/http/view.py", line 146, in handle
homeassistant  |     result = await result
homeassistant  |   File "/usr/src/homeassistant/homeassistant/components/config/config_entries.py", line 180, in post
homeassistant  |     return await super().post(request, flow_id)
homeassistant  |   File "/usr/src/homeassistant/homeassistant/components/http/data_validator.py", line 72, in wrapper
homeassistant  |     result = await method(view, request, data, *args, **kwargs)
homeassistant  |   File "/usr/src/homeassistant/homeassistant/helpers/data_entry_flow.py", line 110, in post
homeassistant  |     result = await self._flow_mgr.async_configure(flow_id, data)
homeassistant  |   File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 271, in async_configure
homeassistant  |     result = await self._async_handle_step(
homeassistant  |   File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 367, in _async_handle_step
homeassistant  |     result: FlowResult = await getattr(flow, method)(user_input)
homeassistant  |   File "/config/custom_components/clever_api/config_flow.py", line 130, in async_step_box
homeassistant  |     resp = await sub.get_evse_info()
homeassistant  |   File "/config/custom_components/clever_api/clever/clever.py", line 201, in get_evse_info
homeassistant  |     model = EvseInfo.parse_obj(resp)
homeassistant  |   File "/usr/local/lib/python3.10/site-packages/pydantic/main.py", line 526, in parse_obj
homeassistant  |     return cls(**obj)
homeassistant  |   File "/usr/local/lib/python3.10/site-packages/pydantic/main.py", line 341, in __init__
homeassistant  |     if validation_error:
homeassistant  | pydantic.error_wrappers.ValidationError: 1 validation error for EvseInfo
homeassistant  | data -> 0 -> smartChargingConfiguration
homeassistant  |   none is not an allowed value (type=type_error.none.not_allowed)
```

**Comment by fars-fede-fire** [comment]
Is this problem fixed in 0.2.3?

**Comment by nvanggaard** [comment]
Yes it is :) 

Thanks a bunch!
